github-actions/src/main/scala/org/typelevel/sbt/gha/WorkflowStep.scala

```diff
@@ -41,16 +41,20 @@ object WorkflowStep {
     UseRef.Public("actions", "checkout", "v3"),
     name = Some("Checkout current branch (fast)"))
 
-  def SetupJava(versions: List[JavaSpec]): List[WorkflowStep] = {
+  def SetupJava(versions: List[JavaSpec], includeSbt: Boolean = true): List[WorkflowStep] = {
```

Comment:
Or maybe `enableSbtCaching` or something

```suggestion
  def SetupJava(versions: List[JavaSpec], enableCaching: Boolean = true): List[WorkflowStep] = {
```